babel/src/main/java/org/apache/calcite/sql/babel/SqlTableAttribute.java

```diff
@@ -20,21 +20,21 @@
 import org.apache.calcite.sql.parser.SqlParserPos;
 
 /**
- * A <code>SqlCreateAttribute</code> is a base class that can be used
+ * A <code>SqlTableAttribute</code> is a base class that can be used
  * to create custom options for the SQL CREATE TABLE function.
  *
  * <p>To customize table option unparsing, override the method
  * {@link #unparse(SqlWriter, int, int)}.
  */
-public abstract class SqlCreateAttribute {
+public abstract class SqlTableAttribute {
   private final SqlParserPos pos;
 
   /**
    * Creates a {@code SqlCreateOption}.
```

Comment:
Should this be "SqlTableAttribute"?

Reply:
I don't think this issue is fixed yet. @dstekol